tests/functional/help.bats

```diff
@@ -21,127 +21,127 @@
 #Based on runc commands
 load common
 
-
 @test "cor -h" {
-  run $COR -h
+  eval run "$COR" -h
```

Comment:
hum could you explain in the commit message what is that needed? could we find a way to not have to eval every time?